Parser.h

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (c) 2021 Chiu Yen-Chen,
+Swen Sun-Yen.
+All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file. See the AUTHORS file for names of contributors.
+ */
+#pragma once
+#include <string>
+#include "RequestData.h"
+
+class Parser
+{
+public:
+	Parser();
+	~Parser();
+
+	RequestData GetRequestData();
```

Comment:
Please add comments for the method, see:
https://google.github.io/styleguide/cppguide.html#Function_Comments